src/main/java/org/folio/rest/impl/BLUsersAPI.java

```diff
@@ -535,6 +538,7 @@ public void getBlUsers(String query, int offset, int limit,
     try {
       userIdResponse[0] = client.request(userUrl.toString(), okapiHeaders);
     } catch (Exception ex) {
+      client.closeClient();
```

Comment:
All lines after `HttpClientInterface client = HttpClientFactory.getHttpClient(okapiURL, tenant);` should be included into the try clause.

Reply:
Not possible as userIdResponse is used after the block.

Reply:
Moved some in there, just now.